docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+    :alt: Add custom role
+    :target: `connect-azure-3.png`_
+    :align: center
+    :figclass: align-center
+
+Name the role and proceed to *Assignable scopes*:
+
+.. _connect-azure-4.png: ../../../_images/connect-azure-4.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-4.png
+    :alt: Name custom role
+    :target: `connect-azure-4.png`_
+    :align: center
+    :figclass: align-center
+
+Ensure the scope is set to the *Subscription* or *Resource Group* you want to use.
+You can change it here if required. Proceed to the JSON tab:
+
+.. _connect-azure-5.png: ../../../_images/connect-azure-5.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-5.png
+    :alt: Review assignable scope
+    :target: `connect-azure-5.png`_
+    :align: center
+    :figclass: align-center
+
+Select *Edit* and replace the *actions* part of the JSON with the one from Hopsworks.ai Azure account configuration workflow.
+Press *Save*, proceed to *Review + create* and create the role:
+
+.. _connect-azure-6.png: ../../../_images/connect-azure-6.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-6.png
+    :alt: Update permission JSON
+    :target: `connect-azure-6.png`_
+    :align: center
+    :figclass: align-center
+
+Step 1.3: Assigning the custom role to Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Back in the *Subscription* or *Resource Group* overview, select *Add* and choose *Add role assignment*:
+
+.. _connect-azure-7.png: ../../../_images/connect-azure-7.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-7.png
+    :alt: Add role assignment
+    :target: `connect-azure-7.png`_
+    :align: center
+    :figclass: align-center
+
+Choose the custom role you just created, select *User, group, or service principal* to *Assign access to*
+and select the *hopsworks.ai* service principal. Press *Save*:
+
+.. _connect-azure-8.png: ../../../_images/connect-azure-8.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-8.png
+    :alt: Configure Hopsworks.ai as role assignment
+    :target: `connect-azure-8.png`_
+    :align: center
+    :figclass: align-center
+
+Go back to the Hopsworks.ai Azure account configuration workflow and proceed to the next step. Copy the CLI command shown:
+
+.. _connect-azure-9.png: ../../../_images/connect-azure-9.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-9.png
+    :alt: Configure subscription and tenant id
+    :target: `connect-azure-9.png`_
+    :align: center
+    :figclass: align-center
+
+Paste the CLI command into your terminal and execute it. Note that you might have multiple entries listed here.
+If so, ensure that you pick the subscription that you want to use.
+
+.. _connect-azure-10.png: ../../../_images/connect-azure-10.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-10.png
+    :alt: Show subscription and tenant id
+    :target: `connect-azure-10.png`_
+    :align: center
+    :figclass: align-center
```

Comment:
Maybe instead of a black square, we should have a fake number, I think it would make things more readable.

Reply:
done